examples/messaging/iterating-multiple-subscriptions/java/Main.java

```diff
@@ -0,0 +1,52 @@
+package example;
+
+import io.nats.client.Connection;
+import io.nats.client.Nats;
+import io.nats.client.Dispatcher;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class Main {
+    public static void main(String[] args) {
+        String natsURL = System.getenv("NATS_URL");
+        if (natsURL == null) {
+            natsURL = "nats://127.0.0.1:4222";
+        }
+
+        // Initialize a connection to the server. The connection is AutoCloseable
+        // on exit.
+        try (Connection nc = Nats.connect(natsURL)) {
+
+            // Create a message dispatcher for handling messages in a
+            // separate thread and then subscribe to multiple target subjects.
```

Comment:
Would you replace the comment with this or something like this:
```
// Create a message dispatcher. A dispatcher a process that runs on
// its own thread, receives incoming messages via a FIFO queue,
// for subjects registered on it. For each message it takes from
// the queue, it makes a blocking call to the MessageHandler
// passed to the createDispatcher call.
```

Reply:
Done